tests/fixtures/tables/antimeridian.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE IF EXISTS antimeridian;
+
+CREATE TABLE antimeridian
+(
+    gid serial PRIMARY KEY,
+    geom GEOMETRY (GEOMETRY, 4326)
```

Comment:
I would prefer if we were to test the other case (`proj == "latlon"` vs `proj_unit == "m"`) as well and rename this testcase with the projection in the name.

Having every case tested makes regressions less likely ^^